modules/postgres/go.mod

```
@@ -6,6 +6,7 @@ require (
 	github.com/docker/go-connections v0.5.0
 	github.com/jackc/pgx/v5 v5.5.4
 	github.com/lib/pq v1.10.9
+	github.com/mdelapenya/tlscert v0.0.0-20240411155228-ea0834bcfa84
```

I released the v0.1.0 this morning.
```suggestion
	github.com/mdelapenya/tlscert v0.1.0
```